tools/rss/pugixml.BUILD

```diff
@@ -0,0 +1,11 @@
+cc_library(
```

Comment:
what do we needs this for?

Reply:
It is also a dependency of Carla Map library, a dependency of Rss
https://ad-map-access.readthedocs.io/en/latest/BUILDING/index.html